### DISCUSSION TITLE
Jestmaster quest bug

### DISCUSSION THREAD

**Discussion by inmanw** [FakeFishGames/Barotrauma#15081] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [X] My issue happened while using mods.

### What happened?

Step 2 of the jestmaster quest will not progress when bonking an abyss creatures. No dialogue pops up and he is not at any outposts. I was using the barotramatic creature pack which has a lot more abyss creatures at first and disabled it to see if that fixes it but did not. 

### Reproduction steps

1 open game
2 kill and bonk abyss creatures 
3 visit clown outpost

### Bug prevalence

Happens every time I play

### Single player or multiplayer?

Single player

### Version

v1.6.19.1 (Unto the Breach Update Hotfix 2)

### -

Windows

### Which operating system did you encounter this bug on?

Windows

**Comment by A2ur3** [comment]
you need to bonk vanilla abyss monster

**Comment by Regalis11** [comment]
Duplicate of https://github.com/FakeFishGames/Barotrauma/discussions/15022, we've got a fix coming up in a future release. Sorry for the inconvenience!
